docs/windows.rst

```diff
@@ -84,7 +84,7 @@ If you want to build a debug version, you must specify ``DEBUG=1`` like below::
 
 From their site: MSYS is a collection of GNU utilities such as bash, make, gawk and grep to allow building of applications and programs which depend on traditional UNIX tools to be present. It is intended to supplement MinGW and the deficiencies of the cmd shell.
 
-MSYS comes in two flavors. The original from MinGW and a MSYS2 http://sourceforge.net/projects/msys2/
+MSYS comes in two flavors. The original from MinGW and an MSYS2 http://sourceforge.net/projects/msys2/
```

Comment:
@k-takata, thank you for the confirmation.

@jwilk, I will fix the sentence I commented here after merging your change.